charts/README.md

```diff
@@ -41,11 +41,69 @@ $ helm uninstall karmada -n karmada-system
 ```
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Example
+### 1. Install agent
+Edited values.yaml
+```YAML
+installMode: "agent"
```

Comment:
I think agent must have both member cluster kubeconfig and karmada kubeconfig. The agent is used to collect member cluster status and sync workload between karmada and member cluster. So if we don't specify member cluster kubeconfig, the agent will have no idea about which cluster it should work with. Note that the script's input have both member cluster kubeconfig and karmada kubeconfig.
https://github.com/karmada-io/karmada/blob/1947cf219979c728f1aa63fd185b33efd821b1de/hack/deploy-karmada-agent.sh#L10

Reply:
You are right. But RBAC is used instead of the kubeconfig. What do you think?

> https://github.com/karmada-io/karmada/blob/1947cf219979c728f1aa63fd185b33efd821b1de/charts/templates/karmada_agent.yaml#L1-L31

Reply:
The RBAC is created in member cluster. Before creating we must know the kubeconfig of member cluster, right?

Reply:
I thought the process should be:
1. Get the kubeconfig of member cluster.
2. Create RBAC in member cluster by member cluster kubeconfig.
3. Deploy karmada-agent, which using RBAC token to watch member cluster and flag `--karmada-kubeconfig` to watch karmada apiserver.

Reply:
Yes, I think I need to add some description. For example:
```console
$ kubectl config use-context member
$ helm install karmada-agent -n karmada-system ./charts
```

Reply:
That's it.👍